confgenerator/logging_receivers.go

```diff
@@ -80,6 +82,15 @@ func (r LoggingReceiverWinevtlog) Type() string {
 	return "windows_event_log"
 }
 
+func (r LoggingReceiverWinevtlog) Components(tag string) []fluentbit.Component {
+	return []fluentbit.Component{
+		fluentbit.WindowsEventlog{
```

Comment:
Just curious. You've replaced a bunch of filters with direct inlined component construction (presumably because they only appear once). This struct also only appears once in the new modular code — was there a particular reason you've kept it? Same question about `Syslog`, actually…

Reply:
That's because structs like `Syslog` and `WindowsEventlog` are shared across the old and new code. Yes, they will both go away as part of removing all the unused code.